source/features/infinite-scroll.tsx

```diff
@@ -33,6 +34,22 @@ function init(): void {
 			inView.observe(button);
 		},
 	});
+
+	// Use cloneNode to keep the original ones for responsive layout
```

Comment:
Turns out this is a wise choice 😁 it works with #5126 flawlessly without changing a single line.